src/class-wpml-beaver-builder-register-strings.php

```diff
@@ -31,8 +31,12 @@ protected function register_strings_for_modules( array $data_array, array $packa
 	 * @return array
 	 */
 	private function sort_modules_before_string_registration( array $modules ) {
-		uasort( $modules, array( $this, 'sort_modules_by_position_only' ) );
-		return $this->sort_modules_by_parent_and_child( $modules );
+		if ( count( $modules ) > 1 ) {
```

Comment:
@strategio Wouldn't it be good if we add a comment here explaining that $modules can be different on frontend editor?

Reply:
@dgwatkins, I think the code is self-explicit, why would we sort if we only have one item?

With this change, we also have a new unit tests which should serve as an explanation (and of course make sure that it fail if we remove this change in the code base).